src/index.js

```diff
@@ -143,3 +145,14 @@ async function setEnvs() {
     env.LOG_LEVEL = (await fs.readFile('envs/watch-keeper-config/LOG_LEVEL', 'utf8')).trim();
   }
 }
+
+process.on('unhandledRejection', (reason) => {
+  log.error(reason, 'Unhandled promise rejection.');
+  process.exit(1);
```

Comment:
this makes me a little nervous. since we run on a cycle, if we have an error thats unhandled, its going to continually restart the pod until we roll out a new version. no?